### PR TITLE
Dev/dagood/pass outputs

### PR DESCRIPTION
**Pull request by mertakman**

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM but will wait for a real engineer to approve before merging :-)

File: eng/pipelines/release-go-images-pipeline.yml
Comment (gdams) [inline eng/pipelines/release-go-images-pipeline.yml:155]: I might be missing something but where is the `generate-announcement` code? Is is this coming in a follow-up PR?

**Pull request closed without merging**
